_pages/en_US/homebrew-launcher-(browserhax-2020).txt

```diff
@@ -0,0 +1,70 @@
+---
+title: "Homebrew Launcher (BrowserHax 2020)"
+---
+
+{% include toc title="Table of Contents" %}
+
+### Required Reading
+
+This method of accessing the Homebrew Launcher takes advantage of some browser oversights in the native 3DS Internet Browser.
+
+If your device already has access to the Homebrew Launcher (whether through BrowserHax, Pichaxx or another exploit), you can skip to [USM](installing-boot9strap-(hbl-usm)). Note that this method requires that your Left and Right Shoulder buttons work.
+{: .notice--success}
+
+### Console Region
+
+To use this guide you will need to know what region your console is operating on.
+
+Note that the console's region can be different than your geographical region - if it is, this does not affect this method.
+
+1. Open System Settings.
+1. Look at the top screen.
+1. In the bottom right of the top screen, you will see a firmware version string, followed by a letter. This letter is your region.
+  + E designates EUROPE.
+  + U designates USA.
+  + J designates JAPAN.
+  + If your console has a letter other than any of these three, it is not compatible with this method.
+1. Remember your region designation. You will need to know this later.
+
+### What You Need
+
+* The latest release of [the Homebrew Launcher](https://github.com/fincs/new-hbmenu/releases/latest)
+* The latest release archive of BrowserHax [[for Old 3DS](https://github.com/zoogie/old-browserhax/releases/latest) / [for New 3DS](https://github.com/zoogie/new-browserhax/releases/latest)]
+  + New/Old refers to the model of your console. If you are unsure of which model you have, consult [this image](https://3ds.eiphax.tech/pic/newold.png)
+* An active WiFi connection on your console
+
+### Instructions
+
+#### Section I - Prep Work
+
+1. Copy `boot.3dsx` from the Homebrew Launcher release to the root of your SD card
+  + The root of the SD card refers to the initial directory on your SD card where you can see the Nintendo 3DS folder, but are not inside of it
+1. Copy `browserhax_hblauncher_ropbin_payload.bin` and `arm11code.bin` from the folder corresponding to your console's region to the root of your SD card
+
+#### Section II - Updating the System
```

Comment:
(This entire section would be removed if integrated into the test at the beginning.)